src/api/system.c

```diff
@@ -12,6 +12,7 @@
   #include <direct.h>
   #include <windows.h>
   #include <fileapi.h>
+  #include <processenv.h>
```

Comment:
I don't think you should include this

Reply:
That’s where SetEnvironmentVariableW is defined though, as per [Microsoft docs](https://learn.microsoft.com/en-us/windows/win32/api/processenv/nf-processenv-setenvironmentvariablew)?
As opposed to the non-w version which is in winbase.

Reply:
processenv.h is included by winbase.h and subsequently included by windows.h